src/emsc_integrator.cpp

```diff
@@ -0,0 +1,108 @@
+#include <iostream>
+#include <sstream>
+#include "emsc_integrator.h"
+
+EmscIntegrator::Exception::Exception(std::string msg, std::string v8_internal_msg)
```

Comment:
As long as you don't move strings inside function, the best solution is to pass strings as const &.